package.json

```
@@ -443,6 +443,26 @@
             }
           }
         },
+        "pyls_mypy": {
+          "title": "MyPy",
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "title": "Enabled",
+              "type": "boolean",
+              "order": 1,
+              "default": true,
+              "description": "Enable or disable MyPy."
+            },
+            "live_mode": {
+              "title": "Live mode",
+              "type": "boolean",
+              "order": 1,
+              "default": false,
```

Without setting this to true it doesn't throw any errors.

```suggestion
              "default": true,
```

---

And yes, we should do that, because that's the default setting for pyls-mypy according to 
https://github.com/tomv564/pyls-mypy#configuration